site/content/en/latest/tasks/traffic/client-traffic-policy.md

```diff
@@ -573,5 +573,26 @@ curl -v http://$GATEWAY_HOST/get \
 request timeout
 ```
 
+### Configure Downstream Per Connection Buffer Limit
+
+This feature allows you to set a soft limit on size of the listener’s new connection read and write buffers.
+The size is configured using the `resource.Quantity` format, see examples [here](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory).
+
+```shell
```

Comment:
we should also make this with two tabs using tabpane like L501, one is `Apply from stdin`, and the other one is `Apply from file`.

Reply:
thanks, fixed both